server/netty-server/cats/src/main/scala/sttp/tapir/server/netty/cats/internal/Fs2StreamCompatible.scala

```diff
@@ -68,6 +66,26 @@ object Fs2StreamCompatible {
       override def emptyStream: streams.BinaryStream =
         fs2.Stream.empty
 
+      override def asWsProcessor[REQ, RESP](
+          pipe: Pipe[F, REQ, RESP],
+          o: WebSocketBodyOutput[Pipe[F, REQ, RESP], REQ, RESP, ?, Fs2Streams[F]],
+          ctx: ChannelHandlerContext
+      ): Processor[WebSocketFrame, WebSocketFrame] = {
+        val processorPromise = ctx.newPromise()
```

Comment:
extract this to a helper method `sendCloseFramePromise`?

Reply:
or rename the value, `processorPromise` doesn't reveal too much about its purpose :)

Reply:
reading the code furhter, I guess this should just be `wsCompletedPromise`?